build.xml

```diff
@@ -517,11 +518,11 @@
   	<target name="build-testware" description=" Build soap harness testware." 
   								depends="clean,build-testware-one,build-testware-two,build-tools-setup,compress-move" />		
 
-		<target name="Run-SoapTestCore" depends="compile" description="Run Staf Tests in non-STAF environment">
-			<property name="testRoot" value="data/soapvalidator/MailClient/Auth/auth_basic.xml"/>
+		<target name="Run-SoapTestCore" depends="compile,zm-network-soap-content" description="Run Staf Tests in non-STAF environment">
```

Comment:
Please create different ant target for zimbraX instead of changing existing one.

Reply:
done